model/schema/core.yaml

```diff
@@ -417,6 +436,15 @@ classes:
       - genome_assembly
       - maps_to_chromosome
 
+  AssemblyComponentDTO:
+    is_a: GenomicEntityDTO
+    description: >-
+      Ingest class for assembly components
+    slots:
+      - display_name
+      - genome_assembly_curie
```

Comment:
The new paradigm for DTO slots referring to classes inheriting from SubmittedObject is to use _identifier as the suffix rather than _curie

Reply:
Fixed

Reply:
We should probably update the description in the slot definition too so that it is clear that a MOD/external curie or internal ID can be used

Reply:
Yep, good point. I've updated the description of mod_entity_id now

Reply:
Sorry, I should have been clearer. I meant the descriptions of the two _identifier slots in this PR - they currently say 'curie', which may be misleading.